pages/api/tabs.ts

```diff
@@ -0,0 +1,16 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { findGuitarTabs } from "../../server/guitarTabs";
+import { withDatabase } from "../../server/utils";
+
+export default withDatabase(
+  async (req: NextApiRequest, res: NextApiResponse) => {
+    const { search } = req.query;
```

Comment:
This solves one problem, but creates another, because this is far less specific than searching for the `category` or `tex`